docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -25,18 +25,7 @@
 //  - attribute: :endpoint-artifact-name-single-sentence-start:
 //      The name of the artifact in singular form with first letter capitalized
 //      Example: Trusted application
-//  - attribute: :endpoint-artifact-api-doc-name:
-//      Hyphenated name for the artifact api doc for section title reference/cross-links
-//      Example: trusted-application
-//  
-//  DOCUMENT IDS:
-//  The reusable content in this file references the following ids which must exist in the document:
 //
-//  - [#create-{endpoint-artifact-api-doc-name}-item-api]
-//  - [#update-{endpoint-artifact-api-doc-name}-item-api]
-//  - [#exception-{endpoint-artifact-api-doc-name}-item-entry-schema]
-//
-//  Note that the section title uses the attribute :endpoint-artifact-api-doc-name for substitution
```

Comment:
This needs to stay. At least that's what @paul-tavares and I agreed on [here](https://github.com/elastic/security-docs/pull/1643#r820806467)

Reply:
@ashokaditya thanks for the heads up, it's added back in: https://github.com/elastic/security-docs/pull/1646/commits/9283d80e8452d295d007ac0077cf8be47b63d7c3